12.0 backend configuration structure

houssine78 opened the pull request.

houssine78 committed:
[IMP] add mail templates on the structure form

houssine78 committed:
[IMP] add ir_mail_server on platform structure

houssine78 committed:
[IMP] overriding easy_my_coop mail template

houssine78 committed:
[IMP] make copy of the overridden

houssine78 committed:
[REFACT] override def retrieving mail template.

houssine78 committed:
[FIX] remove self from function call

houssine78 committed:
[FIX] remove logo from mail templates

houssine78 committed:
[FIX] put menu.xml at the end of the loaded files

houssine78 committed:
[IMP] add board representative and signature scan field

robinkeunen commented:
@houssine78 J'ai rebasé `coopiteasy/ 12.0-backend-configuration-structure` sur 12.0-target. Recommence bien depuis cette branche (celle de coopiteasy) pour ne pas que nous devions pas rerésoudre les conflits plus tard.

```bash
git checkout  12.0-backend-configuration-structure
git reset coopiteasy/ 12.0-backend-configuration-structure --hard
```

Je suggère aussi de laisser @remytms ou moi merger cette branche pour s'assurer qu'on ne merge pas dans 12.0 tout 12.0-target. (cf _12.0-iwp-security commits_ et _merge commits_)

houssine78 committed:
[FIX] fix group name

houssine78 committed:
[IMP] uncomment. problem was due to the loose of the id in vertical coop

robinkeunen commented:
@houssine78 Le dernier commit que t'as pushé me fait un peu peur. Comment ça se fait que tu merges `coopiteasy/12.0-backend-configuration-structure` dans ta branche locale?

houssine78 committed:
[IMP] override certificate report

now we only display shares info from the structure that send the mail.
and we also get info from the coop membership model instead of res
partner

houssine78 committed:
[IMP] override send certificate template.

as we override the report and we change the model of the report from res
partner to subscription register, we give the id of the model instead of
the id of the res partner.

houssine78 committed:
[IMP] remove email template field on product form

houssine78 committed:
[IMP] change the model of the email template certificate.

as we need to filter the share on the structure  the report on the
certificate report, we changed the model from res_partner to
subscription register to allow this

houssine78 committed:
[FIX] fix function name typo

houssine78 committed:
[IMP] add computed fields on coop membership

remytms commented:
@robinkeunen @houssine78 Je ferai un rebase sur cette branch quand @houssine78 sera de retour. Ou s'il est vraiment nécessaire de la mettre en test tout de suite, car si je réécris l'historique maintenant ça va poser des problème à Houssine.

Il faudra retirer les commits suivant de cette PR :

[REF] iwp: consistent group naming
451997c

[REM] iwp: remove ir rules
7bbcf05

[IMP] review ir model access
5ae4859

[ADD] demo data and test subscription flow
06c4877

[REM] loan term rules and menus
592cbea

[FIX] line is breaking installation
e1d14dc

[ADD] ir.rules, demo and test
b16ee89

[ADD] iwp: user access to own structure only
77c71e5

fix tests
c4c863c

[REF] use user.structure in rules
7f8ae06

[ADD] iwp_base: Stat about financial products and outstanding
fb1c9a6

[ADD] iwp_base: Creation mode on share lines
634c7cc

[ADD] iwp_website: Creation mode in share wallet
be54a0f

[ADD] iwp_website: Form to create manual share line
b64f42e

[ADD] iwp_website: Action to delete manual share line
7566d00

[FIX] iwp_website: Show monetary field correctly
5a562a7

[ADD] iwp_base: Filter for structure
d9d6a11

[ADD] iwp_website: Button to take manual share
ad27c0f

TO BE DELETED !! [ADD] iwp_website: Action to delete manual share line
fee4462

houssine78 committed:
[IMP] override functions for mails and adding structure in vals

houssine78 committed:
[IMP] add dedicated certificate reports and make mail templates use them

houssine78 commented:
J'avais fait un rebase mais je croix que mon plugin éclipse fait de la
merde. Je vais commencer à faire ça en cmd line ;)

Le lun. 25 nov. 2019 à 12:31, Robin Keunen <notifications@github.com> a
écrit :

> @houssine78 <https://github.com/houssine78> Le dernier commit que t'as
> pushé me fait un peu peur. Comment ça se fait que tu merges
> coopiteasy/12.0-backend-configuration-structure dans ta branche locale?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coopiteasy/investor-wallet-platform/pull/33?email_source=notifications&email_token=AB3IE6VYACOYKZWN6K43LUDQVOZQXA5CNFSM4JQJGWH2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEFCCT2I#issuecomment-558115305>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB3IE6WES4EKAS7XEAUBZF3QVOZQXANCNFSM4JQJGWHQ>
> .
>

houssine78 committed:
[FIX] fix security group in form view
